aggsender/types/block_range.go

```diff
@@ -70,3 +74,11 @@ func getBlockMinusOne(fromBlock uint64) uint64 {
 	}
 	return 0
 }
+
+func (b BlockRange) IsNextContigousBlock(other BlockRange) bool {
+	// Check if the two ranges are contiguous
+	if b.ToBlock+1 == other.FromBlock || other.ToBlock+1 == b.FromBlock {
+		return true
+	}
+	return false
```

Comment:
Small simplification, wdyt?
```suggestion
        return b.ToBlock+1 == other.FromBlock || other.ToBlock+1 == b.FromBlock
```

Reply:
I have change a bit the function to be more strict and also apply your simplification